coordinator/core/core.go

```diff
@@ -38,25 +39,15 @@ import (
 
 // Core implements the core logic of the Coordinator
 type Core struct {
-	rootCert          *x509.Certificate
-	intermediateCert  *x509.Certificate
-	adminCerts        []*x509.Certificate
-	quote             []byte
-	rootPrivK         *ecdsa.PrivateKey
-	intermediatePrivK *ecdsa.PrivateKey
-	sealer            Sealer
-	recovery          recovery.Recovery
-	manifest          manifest.Manifest
-	rawManifest       []byte
-	updateManifest    manifest.Manifest
-	rawUpdateManifest []byte
-	secrets           map[string]manifest.Secret
-	state             state
-	qv                quote.Validator
-	qi                quote.Issuer
-	activations       map[string]uint
-	mux               sync.Mutex
-	zaplogger         *zap.Logger
+	quote          []byte
+	recovery       recovery.Recovery
+	manifest       manifest.Manifest
+	updateManifest manifest.Manifest
```

Comment:
Why are we keeping `manifest` and `updateManifest` in the Core? They just keep duplicate values to the ones in the store, no?

Reply:
Its more of a convenience thing than anything else. Since the manifest does not change after it has been set once, it is kept as a `manifest.Manifest` object in memory. This removes the need to load the manifest from the store every time we want to iterate over values set by the manifest (for example when generating secrets for a marble).

We can of course change this to instead fetch it just in time using the `storeWrapper` `getManfiest` method

Reply:
Yeah, I'd keep it simple and omit caching for now. Considering future changes and potentially a distributed Coordinator we should refrain from special cases at this point.

Reply:
It may be best to parse the manifest once and insert the values into the store. But does not need to happen within this PR.

Reply:
We can then also get rid of the update manifest.